stackslib/src/main.rs

```diff
@@ -1645,7 +1681,12 @@ fn replay_block(stacks_path: &str, index_block_hash_hex: &str) {
         return;
     };
 
-    let block = StacksChainState::extract_stacks_block(&next_staging_block).unwrap();
+    let epoch_id = SortitionDB::get_stacks_epoch(&sort_tx, u64::from(burn_header_height))
```

Comment:
No need for this -- the block has already been processed so there's no need to revalidate it with the epoch